### ISSUE TITLE
Follow guidelines to structure a Flask project

### ISSUE THREAD

**Issue by rgreinho** [dsmurl/SystemController4#45]
As the application will grow, it would be best to start using the best practices for structuring the project. This will definitely help in automating tasks like tests and deployment, as well as bringing clarity to the project. The official documentation is here: http://flask.pocoo.org/docs/0.10/patterns/packages.

There are also other resources that may be useful:
- http://mattupstate.com/python/2013/06/26/how-i-structure-my-flask-applications.html
- https://www.digitalocean.com/community/tutorials/how-to-structure-large-flask-applications

**Comment by dsmurl** [comment]
Cool, I'll have to read through these and then maybe this issue can be turned into a parent issue where we can collaborate to make a solid, defined list of preferred changes.

**Comment by rgreinho** [comment]
Agreed :) Also, `Flask Blueprints` should be used to create modular components: http://flask.pocoo.org/docs/0.10/blueprints.
